examples/pipelines/datacomp/pipeline.py

```diff
@@ -41,70 +40,70 @@
         "dataset_name": "nielsr/datacomp-small-with-text-embeddings",
         "column_name_mapping": load_component_column_mapping,
         "index_column": "uid",
-        # "n_rows_to_load": 1000,
+        "n_rows_to_load": 1000,
+    },
+    node_pool_label="node_pool",
+    node_pool_name="n2-standard-64-pool",
+)
+download_images_op = ComponentOp.from_registry(
+    name="download_images",
+    arguments={
+        "retries": 2,
+        "min_image_size": 0,
+    },
+    node_pool_label="node_pool",
+    node_pool_name="n2-standard-64-pool",
+    input_partition_rows=-1,
+    cache=False,
+)
+detect_text_op = ComponentOp(
+    component_dir="components/detect_text",
+    arguments={
+        "batch_size": 2,
+    },
+    node_pool_label="node_pool",
+    node_pool_name="model-inference-mega-pool",
```

Comment:
```suggestion
    node_pool_name="model-inference-pool",
```
A100s are not really available atm